src/apps/activities/domain/scores_reports.py

```diff
@@ -181,6 +185,10 @@ def validate_raw_score_lookup(cls, value):
     def validate_score_lookup(cls, value):
         return validate_score_subscale_table(value)
 
+    @validator("age")
+    def validate_age_lookup(cls, value):
```

Comment:
if you add parameter `pre=true` to @validator(), it will validate before assign value to age, so age don't need to be `str`

Reply:
Oh, thanks for the tip. I made it a `str` because it will need to store the range value with the tilde (~) in it. However, I just made another change that adds back `PositiveInt` so the field remains backwards compatible because I realised this was a breaking change